Test/linters/constructorCaseWithoutParenthesis.dfy.expect

```diff
@@ -0,0 +1,11 @@
+constructorCaseWithoutParenthesis.dfy(9,6): Warning: this branch is redundant 
+constructorCaseWithoutParenthesis.dfy(10,6): Warning: this branch is redundant 
+constructorCaseWithoutParenthesis.dfy(15,6): Warning: this branch is redundant 
+constructorCaseWithoutParenthesis.dfy(16,6): Warning: this branch is redundant 
+constructorCaseWithoutParenthesis.dfy(24,9): Warning: this branch is redundant 
+constructorCaseWithoutParenthesis.dfy(25,9): Warning: this branch is redundant 
+constructorCaseWithoutParenthesis.dfy(9,11): Warning: Constructor name 'Green' should contain a parenthesis at the end
```

Comment:
I think you want to use `parentheses` instead of `a parenthesis`, to be plural instead of singular. More information: https://www.grammarbook.com/blog/commas/all-about-parentheses/#:~:text=The%20singular%20form%20is%20parenthesis,So%20let's%20keep%20it%20simple.

In particular, it says: "a parenthesis is one of a pair of curved marks that look like this: ( ), and parentheses are both marks."

Also, I find "at the end" is a little ambiguous. The end of what? And I think we can be more specific about when this warning occurs.

Something like "A constructor name in a case pattern should be followed by parentheses."